packages/core/src/products/listing/redux/reducer.js

```diff
@@ -97,16 +105,41 @@ export const entitiesMapper = {
   },
 };
 
+const listingFacets = (state = INITIAL_STATE.listingFacets, action = {}) => {
+  switch (action.type) {
+    case GET_LISTING_FACETS_REQUEST:
+      return {
+        ...state,
+        isLoading: true,
+      };
+    case GET_LISTING_FACETS_SUCCESS:
```

Comment:
I think `isLoading` set to false(*), is missing in success or failure actions.

(*) could be set by using default initialState value which is false.

Reply:
Fixed!

Reply:
I think might be easier do like that:
`
{
        ...INITIAL_STATE.listingFacets,
         result: action.payload.result,
 }
`

Same for following returns